mmdet/models/detectors/yolact.py

```diff
@@ -25,12 +25,24 @@ def __init__(self,
         self.segm_head = build_head(segm_head)
         self.mask_head = build_head(mask_head)
 
-    def forward_dummy(self, img):
+    def forward_dummy(self, imgs):
         """Used for computing network flops.
 
         See `mmdetection/tools/analysis_tools/get_flops.py`
         """
-        raise NotImplementedError
+        # img_metas do not influence the flops calculation
+        dummy_img_metas = [
+            dict(
+                img_shape=img.shape, ori_shape=img.shape, scale_factor=(1, 1))
+            for img in imgs
+        ]
+        feat = self.extract_feat(imgs)
+        bbox_outs = self.bbox_head(feat)
+        bboxes = torch.rand((1, 1, 4)).to(imgs.device)
```

Comment:
This number cannot be set to 1 will cause inaccurate calculations, and should be the same as the actual output number.

Reply:
It seems that `bboxes` passed to `forward` of `mask_head` is used for cropping the results, so it does not influence the calculation of flops. I have experimented with different shape of `bboxes` and the results are all the same. Or should the cropping of the results also be counted as flops? 
Actually the part of `mask_head`'s forward really contains the forwarding of a network is
```
prototypes = self.protonet(x)
```
Can I create a `forward_dummy` for the `mask_head` that just calls the `protonet`? In this way we do not need to generate `img_meta`, `bboxes` and `det_coeffs`. The problem may be what we output from `forward_dummy` is different from that of `simple_test`, should the outputs of `forward_dummy` represent anything?

Reply:
@hhaAndroid

Reply:
@collinzrj Just follow `forward_dummy` not including the post-processing part.